checker/src/org/checkerframework/checker/lock/LockStore.java

```diff
@@ -22,16 +27,23 @@
  */
 public class LockStore extends CFAbstractStore<CFValue, LockStore> {
 
+    /** If true, indicates that the store refers to a point in the code
+      * inside a constructor or initializer. This is useful because
```

Comment:
I assume initializer only means instance initializers. Is there anything special for static initializer blocks?